modules/stepper/demo/stepper.html

```diff
@@ -73,6 +73,12 @@ <h1 class="main-section__title">Stepper</h1>
                     <td><code>horizontal</code></td>
                     <td>Define the stepper layout. Options: <code>horizontal</code>, <code>vertical</code></td>
                 </tr>
+                <tr>
+                    <td>lx-contols</td>
```

Comment:
Just to avoid confusion, I would have called it lx-inner-controls rather than just lx-controls just because in the code if I see controls = false I'm tempted to think we are just not displaying the controls at all.